recipes/antlr/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+    name: antlr
+    version: 2.7.7
+
+source:
+    fn: antlr-2.7.7.tar.gz
+    url: http://www.antlr2.org/download/antlr-2.7.7.tar.gz
+    patches:
+        - CharScanner.patch
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - python >=2.7,<3
+    run:
+        - python
+
+test:
+    commands:
+        - antlr --help
+
+about:
+    home: http://www.antlr2.org/
+    license: http://www.antlr2.org/license.html
+    summary: ANTLR, ANother Tool for Language Recognition, 2.7.7
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - kwilcox
```

Comment:
Are recipe-maintainers automatically added to the feedstock repository?  I didn't see this with the recipes I recently added but I may have done something wrong.